kubernetes/opentelemetry-demo.yaml

```diff
@@ -9910,7 +9910,7 @@ spec:
       serviceAccountName: opentelemetry-demo
       containers:
         - name: flagd
-          image: 'ghcr.io/open-feature/flagd:v0.11.1'
+          image: 'ghcr.io/open-feature/flagd:v0.11.2'
```

Comment:
Would you mind reverting this change here?

This file should be updated only with:

```
make generate-kubernetes-manifests
```

We usually run it when there is a new release and helm update.
https://github.com/open-telemetry/opentelemetry-demo/blob/main/Makefile#L114-L126

Reply:
@Starefossen would you be able to revert this one so we can get this merged?